src/sentry/api/endpoints/group_user_reports.py

```diff
@@ -19,7 +20,18 @@ def get(self, request, group):
         :auth: required
         """
 
-        report_list = UserReport.objects.filter(group=group)
+        try:
+            environment = self._get_environment_from_request(
+                request,
+                group.organization.id,
+            )
+        except Environment.DoesNotExist:
+            report_list = Environment.objects.none()
+        else:
+            if environment is not None:
```

Comment:
We can rewrite this as:

```python
report_list = UserReport.objects.filter(group=group)
if environment is not None:
    report_list = report_list.filter(environment=environment)
```

Since you can chain filters.

Reply:
I'd seen this before. I'd just forgotten that was something I could do. Thanks! :)